specifications/Privateform-FRD.md

```diff
@@ -0,0 +1,779 @@
+# Privateform - Functional Requirements Document
```

Comment:
Can we add some sort of a delimiter to better separate the sections, for example a dotted line at the end of each section. Or maybe somehow better outline the headlines? Right now scrolling the docs it's hard to notice when a section ends and a new starts.

Reply:
I added dotted line after each section for both documents